src/editors/containers/TextEditor/components/ErrorAlerts/FeedbackAlert.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Alert } from '@edx/paragon';
+import { Info } from '@edx/paragon/icons';
+
```

Comment:
I created this instead of using ErrorAlert.jsx because of the Alert.Heading.

Reply:
plz just make the error title and thus heading optional in the ErrorAlert rather than repeating code.

Reply:
This makes sense. Will be coding it in